packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py

```diff
@@ -263,28 +263,37 @@ def wait_for_pipeline_state(
     return current_state
 
 
-def on_web_socket_default_handler(ws) -> None:
-    """Usage
-
-    from pytest_simcore.playwright_utils import on_web_socket_default_handler
-
-    page.on("websocket", on_web_socket_default_handler)
+@contextlib.contextmanager
+def web_socket_default_log_handler(web_socket: WebSocket) -> Iterator[None]:
 
-    """
-    stack = ExitStack()
-    ctx = stack.enter_context(
-        log_context(
+    try:
+        with log_context(
             logging.INFO,
-            (
-                f"WebSocket opened: {ws.url}",
-                "WebSocket closed",
-            ),
-        )
-    )
-
-    ws.on("framesent", lambda payload: ctx.logger.info("⬇️ %s", payload))
-    ws.on("framereceived", lambda payload: ctx.logger.info("⬆️ %s", payload))
-    ws.on("close", lambda payload: stack.close())  # noqa: ARG005
+            msg="handle websocket message (set to DEBUG level if you wanna see all of them)",
+        ) as ctx:
+
+            def on_framesent(payload: str | bytes) -> None:
+                ctx.logger.debug("⬇️ %s", payload)
+
+            def on_framereceived(payload: str | bytes) -> None:
+                ctx.logger.debug("⬆️ %s", payload)
+
+            def on_close(payload: WebSocket) -> None:
+                ctx.logger.warning("Websocket closed: %s", payload)
```

Comment:
no emoticos for close or error? :-(

- **Closed**: 🚪 (Door)
- **Error**: ❌ (Cross Mark)

Reply:
where do you get those?